### ISSUE TITLE
-1 argument downloads the older episodes

### ISSUE THREAD

**Issue by rohshall** [mlj/castget#26]
I am subscribed to some news podcasts, and I prefer to have only the latest episodes. So, castget -1 should get only the latest episode, if it exists. However, every time I run castget -1, it downloads one older episode. It should check the latest enclosure, and if the file already exists, and do nothing, instead of downloading the older episode.

**Comment by mlj** [comment]
I agree that this is something we'd want, but I think we should keep `-1` the way it is and add this as a new option or as the possible outcome of a combination of options.
